projects/plugins/backup/jetpack-backup.php

```diff
@@ -100,6 +100,26 @@ function () use ( $jetpack_backup_meets_requirements ) {
 		);
 	}
 
+	// Add a red bubble notification to My Jetpack if the installation is bad.
+	add_filter(
+		'my_jetpack_red_bubble_notification_slugs',
+		function ( $slugs ) {
+			$red_bubble_notificationd_data = array(
+				'jetpack-vaultpress-backup-plugin-bad-installation' => array(
+					'data' => array(
+						'plugin' => 'Jetpack VaultPress Backup',
+					),
+				),
+			);
+
+			if ( ! is_array( $slugs ) ) {
+				return $red_bubble_notificationd_data;
+			}
```

Comment:
I'm not sure this check is necessary...?  Is it possible that the `$slugs` argument will ever not be an array?
Looking at where the `apply_filters( 'my_jetpack_red_bubble_notification_slugs', $red_bubble_alerts );` is called (class-initializer.php, line 652), it looks to me that `$red_bubble_alerts` is first initialized as a static array() before the apply_filters call. Unless I'm missing something or misunderstanding how it works..🤔?

Reply:
I think you are right 🤔 

I was getting some "$red_bubble_alerts is not an array" errors early on, but it seems as though that's probably just due to something I was doing wrong at the time